### ISSUE TITLE
Colour swatches for single product

### ISSUE THREAD

**Issue by iamdanbarrett** [CrocoBlock/suggestions#361]
It would be great to have a widget that would allow us to design a colour swatch palette for products...

**Comment by Floydtm** [comment]
@iamdanbarrett, hi! 
Our team will consider your proposal. In the future we will try to implement it. For now you can find a solution [here](https://wordpress.org/plugins/search/colour+swatch/).

**Comment by iamdanbarrett** [comment]
This is what I have had to implement in the interim - I am hoping you guys will be able to add this to your Woo builder because it would be widely used.

**Comment by michalczajka** [comment]
Hope You will make it soon to use it on archive page ;)

**Comment by iamdanbarrett** [comment]
Was there ever a release on this?

**Comment by iamdanbarrett** [comment]
Any news on this at all?

**Comment by noxdiurna** [comment]
I suppose that this functionality is yet to be placed at the priority but I wish it would as adding other plugins would work on individual product page but it won't on JetWoo Product Grid.

**Comment by adelapuentep** [comment]
Please help us with this JetWooBuilder feature for the Single Product Page. JetSmartFilters already had it implement for the Shop/Archive page, it's kind of weird not to allow it within JetWooBuilder.

**Comment by sara-kariv** [comment]
is there an ETA on adding variation swatches on jet woo product grid?

**Comment by fabi20x** [comment]
any timeline when the variation in grid listing might be added? strong need for that feature

**Comment by adelapuentep** [comment]
Until Crocoblock adds it as an official feature, they've created this guide:
https://crocoblock.com/troubleshooting/articles/how-to-add-attractive-variation-swatches-to-woocommerce-archive-page/

Haven't try it yet but will do it.

**Comment by UraraReika** [comment]
Right now variation swatches works in single product templates and you can use whatever you want swatches plugin. Because our functionality based on WooCommerce. But for product cards we wrote  this guide.

**Comment by sara-kariv** [comment]
> Until Crocoblock adds it as an official feature, they've created this guide:
> https://crocoblock.com/troubleshooting/articles/how-to-add-attractive-variation-swatches-to-woocommerce-archive-page/
> 
> Haven't try it yet but will do it.

I've tried that guide and the codes there crashed my website...

**Comment by fabi20x** [comment]
Yes mine too!

**Comment by adelapuentep** [comment]
Yes, something from this guide is not detailed correctly and it seems to be on the Step 5 when you edit the "item-button" file.

**Comment by Cheezitcrackers** [comment]
Website crashed after following the guide and inserting the code into functions.php

**Comment by UraraReika** [comment]
There is some problem in the article, sorry for that. Here https://prnt.sc/zkv13i is wrong link. The right link is https://wordpress.org/plugins/woo-variation-swatches/ we will change it as soon as possible.

**Comment by adelapuentep** [comment]
> There is some problem in the article, sorry for that. Here https://prnt.sc/zkv13i is wrong link. The right link is https://wordpress.org/plugins/woo-variation-swatches/ we will change it as soon as possible.

You will see on the comment's guide that I personally challenged (with humble) Crocoblock's team to follow the guide step by step so they could also see the mistake and the said that they did and everything was working fine. I have another question, on the same screenshot that you just uploaded at the end of the plugin's name it says "pro" but it isn't underlined. Do we need to use the pro version of the plugin to make it work? And lastly, PLEASE, once you update the guide refollow it STEP BY STEP and let us know for real that it's working. Maaaybe, after all these mistakes, an EASY video tutorial could really help. Thank you for the support @UraraReika.

**Comment by UraraReika** [comment]
Thank you @adelapuentep ! I'll reread and recreate step by step thus guide, then will correct the article if something is wrong!  Although i'll think about short video.

**Comment by neurohost** [comment]
It would be great if this function is applied natively in jetwoobuilder wigets, without the need to edit the code or third-party plugins, please crocoblock team, this function is of vital importance for ecommerce

**Comment by k4znIm** [comment]
Agree with that, thats really needed functionality. It woud be nice to have option change and design variation lists like colors, sizes, etc.. Is there any ETA when we can expect that?

**Comment by getwebcare** [comment]
+1 and also image variations would be nice.

**Comment by SMolodoy** [comment]
+1

**Comment by juanisidoro** [comment]
adding another interested +1.
THANKS!

**Comment by 79ho3ein** [comment]
+1

**Comment by 79ho3ein** [comment]
@UraraReika 
Hello, will this feature be added to jet woobuilder in the future?
Or has it been canceled?
thanks

**Comment by 79ho3ein** [comment]
hi @Floydtm 
excuse me... Does jet woobuilder have a plan to add this important feature to the plugin?
thanks

**Comment by 79ho3ein** [comment]
hello @UraraReika
It is now 2 years since this request was opened. Does jet woobuilder currently have a plan to add this important feature to the plugin?
Can we hope to see this feature in jet woobuilder in 2021?

thanks you

**Comment by 79ho3ein** [comment]
This image is to introduce the jet woobuilder plugin on the crocoblock website.
![Screenshot (57)](https://user-images.githubusercontent.com/77163248/123540379-e63dcf80-d753-11eb-91db-d6e796f9fd16.png)
As you can see in the image, the variables section, color selection and size are superbly displayed. These features in the image are not currently available in this plugin.

But if this feature is added to the plugin exactly as it is in the image, it will be great. Because it is excellent and well designed.
The current variable selection section of the plugin is very old. 
The jet woobuilder plugin is incredibly powerful and I personally have not seen a better plugin than jet woobuilder. But it is a pity that these features in the image above are not available in the plugin.
**I hope jet woobuilder will add these features to its plugin soon.**

**Comment by Lonsdale201** [comment]
Any news about it? :)

**Comment by k4znIm** [comment]
I cant wait to see thta in croco.. I really need that :D

**Comment by 79ho3ein** [comment]
@Lonsdale201
I do not think they will add this feature. Because they have already suggested using a third party plugin.
This means that in order to have the "Color swatches" feature, we have to install 2 other plugins in addition to the jet woobuilder. And this does not make sense at all.
In any case, I hope to add this feature to the jet woobuilder core. This feature is extremely useful and required.

**Comment by k4znIm** [comment]
@79ho3ein 

I know they told they will not be adding features what u can add by another plugin. But this doesnt make sense.. Why i have to install 2 more plugins, if this can be part of jetWooBuilder? I have crocoblock best licence.. but i think more people will decide to buy crocoblock if they have all these functionalities in one plugin.

**Comment by 79ho3ein** [comment]
@k4znIm 
Exactly. jet woobuilder is a very large plugin, but it is very surprising that its developers delegate such an important feature to a third-party plugin.
I can boldly say that 99% of users of this plugin want this important feature to be added to the plugin. But it does not really make sense to install 2 plugins to install this feature. **Because everyone knows that the most important thing in maintaining an optimal site is to reduce the number of plugins.**

**Comment by k4znIm** [comment]
@79ho3ein 

"Because everyone knows that the most important thing in maintaining an optimal site is to reduce the number of plugins."

Yes, this it exactly what i mean. I hope the will add this.. Its weird if they will not add it and styll have it in plugin promo image..

**Comment by 79ho3ein** [comment]
@k4znIm
Surely they will add this feature soon. Of course I hope!
Of course The fact that developers do not respond to this request does not mean that they ignore the request of their users.
Incidentally, in my opinion, jet woobuilder has the best and most powerful developers. They pay more attention to their users' requests than any other developer. This is quite clear and I thank them.
Finally, I hope to see this feature in Jet woobuilder soon.

thank you

**Comment by k4znIm** [comment]
@79ho3ein 
Thats true, all crocoblock team members are really helpfull and they are listening theyrs users!

**Comment by mehdimoradi1500** [comment]
+1 and also images based selection for colors

**Comment by 93petter93** [comment]
+ 1

**Comment by 93petter93** [comment]
Color swatches and product size swatches would be amazing.

**Comment by mehdimoradi1500** [comment]
Hi crocoblock team
Why it takes so long to add a "new necessary feature" to jet woo builder?
There are so many requests for these necessary features that every simple theme has them, but unfortunately jet woo builder doesn't? 
So please respect customers and make these widgets faster
Thank you

**Comment by 79ho3ein** [comment]
> Color swatches and product size swatches would be amazing.

Yes, this feature is amazing. But unfortunately I do not think they have any plans to add this feature to the jet woobuilder core.
**It is very strange ...**

**Comment by gokhanbelli** [comment]
> Until Crocoblock adds it as an official feature, they've created this guide: https://crocoblock.com/troubleshooting/articles/how-to-add-attractive-variation-swatches-to-woocommerce-archive-page/
> 
> Haven't try it yet but will do it.

I'm using woodmart theme, but it didn't work

**Comment by adelapuentep** [comment]
Please Crocoblock team, add this feature as a priority. It's been more than 2 years now since this thread was opened and even though you've taken the time to create a guide to achieve this with a third party plugin, it's very restrictive since we have to use Kava theme and paid additionally for the pro version of the swatches plugin. Help us keep developing awesome websites without the need of additional plugins (and potentially broken sites or incompatible updates).

**Comment by garenian1** [comment]
Please add this feature so we can keep using your set of plugins instead of pushing us away from crocoblock to use other company's plugins.

**Comment by 79ho3ein** [comment]
Is there any news about this feature?
We are nearing the end of 2021 and it is unlikely to be published this year. But I hope it will be added in 2022 ...

**Comment by chicoco07** [comment]
+1

**Comment by Joliamendi** [comment]
Dear developers of jet woobuilder. Is there any news of this request? Please if you have any news

**Comment by subedinfo-it** [comment]
+1 for this request. Something alike woocommerce product swatches

**Comment by scottybo** [comment]
+1

**Comment by awebguy99** [comment]
+1 I've just started using various Jet plugins and was very surprised the color swatches isn't available on the product page.

**Comment by Michaelgimii** [comment]
Hi Dear Crocoblock developers @Floydtm @UraraReika  @Crocoblock 

These days you release big updates for your plugins and that's really great. Considering that you are living in the days of making war...
So, do you have any plans to add color swatches to jet woobuilder? This feature is one of the most popular requests from your users on GitHub. I hope you will respond if you are reading this. Even if the answer is no. We are really looking forward to the release of this great feature.

best regards

**Comment by Joliamendi** [comment]
I installed and tested most of the color swatches plugins. Almost all of them had serious interference with jet woobuilder.I hope it will be added to jet woobuilder.

**Comment by Mrgreen93** [comment]
The same situation for me, clear roadmap will help so many users and solve their concerns 🌷🙏

**Comment by eitayz** [comment]
Waiting also for that feature. Will help many Crockos out there.

**Comment by llowboy01** [comment]
I'm doing a bit of research before buying a license. I'm really amazed by this missing feature. This will make me not buying the license at the moment because this is essential for me!

**Comment by buithucdong** [comment]
We are still waiting for this feature to be developed every day

**Comment by razkoko1** [comment]
+1 its a must feature for woocommerce plug in.
Plz add it ASAP

Thanks!

**Comment by Michaelgimii** [comment]
Still no news about this feature?
I wish there was an answer for us, even if the answer is no. Does anyone from crocoblock team see here? This feature is the most popular request on GitHub, but it is never taken care of.

**Comment by Robinsgood** [comment]
+1

**Comment by eitayz** [comment]
I wrote them via support chat asking them to tell us about their plans for this feature.

**Comment by eitayz** [comment]
This the answer I got from Crocoblock:
"Hi there!
The developers work on requests on GitHub in the order they determine themselves (depending on different factors)."

I think more users just need to upvote this feature.
They do a great job in my opinion, hope they will hear our voice :)

**Comment by Marcoqwerty78** [comment]
Mandatory the Variation Swatches Widget, most of the online plugin are not compatible with JetWooBuilder; workaround require a lot of code customization and become less flexible.
Im getting mad for this.
